pinc/ProofreadingToolbox.inc

```diff
@@ -33,7 +33,11 @@ class ProofreadingToolbox
     {
         echo "<div class='display-flex' id='toolbox'>";
         $character_selector = new CharacterSelector;
+        echo "<div id='char-picker'>";
+        echo "<button type='button' id='hide-picker' class='hide-picker-button' title='", _("Hide the character picker"), "'>&lang;</button>";
```

Comment:
Please wrap this in `html_attr()`: `html_attr(_("Hide the character picker"))`. Ditto the show string.